clonevirtualenv.py

```diff
@@ -108,7 +109,7 @@ def fix_symlink_if_necessary(src_dir, dst_dir):
                 target = os.path.realpath(full_file_path)
                 if target.startswith(src_dir):
                     new_target = target.replace(src_dir, dst_dir)
-                    logger.debug('fixing symlink in {}'.format(full_file_path))
+                    logger.debug('fixing symlink in %s' % full_file_path)
```

Comment:
Why is this needed?

Reply:
Originally the existing tox.ini file included testing python2.6. 2.6 did not accept empty format directives '{}', so I rewrote them in "portable", 2/3 independent way.  This way it also works for 2.6 even if I later changes the tox.ini to match the travis config which does not test 2.6... This is perfectly legal and accepted python 3...